README.md

```diff
@@ -17,13 +17,13 @@ If a crate doesn't have this field, no README will be displayed.
 Docs.rs is using semver to parse URLs. You can use this feature to access
 crates' documentation easily. Example of URL redirections for `clap` crate:
 
-| URL                          | Redirects to documentation of                  |
-|------------------------------|------------------------------------------------|
-| <https://docs.rs/clap>       | Latest version of clap                         |
-| <https://docs.rs/clap/~2>    | 2.* version                                    |
-| <https://docs.rs/clap/~2.9>  | 2.9.* version                                  |
-| <https://docs.rs/clap/2.9.3> | 2.9.3 version (you don't need = unlike semver) |
-
+| URL                                           | Redirects to documentation of                    |
+|-----------------------------------------------|--------------------------------------------------|
+| <https://docs.rs/clap>                        | Latest version of clap                           |
+| <https://docs.rs/clap/~2>                     | 2.* version                                      |
+| <https://docs.rs/clap/~2.9>                   | 2.9.* version                                    |
+| <https://docs.rs/clap/2.9.3>                  | 2.9.3 version (you don't need = unlike semver)   |
+| <https://docs.rs/clap/*/clap/struct.App.html> | Latest version of this page (if it still exists).|
 The crates.fyi domain will redirect to docs.rs, supporting all of the
```

Comment:
This puts this line inside of the table instead of outside. However, `crates.fyi` is no longer running so I think we can just remove the line altogether.